Views/BuildingBlocks/WebView.swift

```diff
@@ -108,6 +119,12 @@ class WebViewController: UIViewController {
                 self?.topSafeAreaConstraint?.isActive = false
                 self?.view.topAnchor.constraint(equalTo: webView.topAnchor).isActive = true
             }
+        
+        if parent?.navigationController != nil {
+            compactViewNavigationController = parent?.navigationController
+        } else {
+            debugPrint("compactViewNavigationController not set")
```

Comment:
Please change (all) ``debugPrint`` to ``os_log`` you can also consider adding a new ``subsystem`` in the `struct Log` for ``WebView``.

Reply:
Thanks, I've updated as per your recommendation.